core/pom.xml

```diff
@@ -11,6 +11,11 @@
   <properties>
     <guava.version>20.0</guava.version>
     <gwt.version>2.8.0</gwt.version>
+    <junit.version>4.12</junit.version>
+    <jsr305.version>3.0.1</jsr305.version>
+    <auto-value.version>1.4.1</auto-value.version>
+    <compile-testing.version>0.10</compile-testing.version>
```

Comment:
Yeah, at this point, I believe everyone is going to build on 8, even if they run on 7/6. And now that Jack/Jill is being deprecated in favor of java8 desugarring for android, I don't even think it'll ever be an issue if we can't test in 8.